src/server/pfs/server/driver.go

```diff
@@ -533,44 +616,48 @@ func (d *driver) canDeleteRepo(txnCtx *txncontext.TransactionContext, repo *pfs.
 }
 
 func (d *driver) createProject(ctx context.Context, req *pfs.CreateProjectRequest) error {
+	return d.txnEnv.WithWriteContext(ctx, func(txnCtx *txncontext.TransactionContext) error {
+		return d.createProjectInTransaction(txnCtx, req)
+	})
+}
+
+func (d *driver) createProjectInTransaction(txnCtx *txncontext.TransactionContext, req *pfs.CreateProjectRequest) error {
```

Comment:
I'm curious to know, is there a reason why we aren't doing this in a write transaction anymore?

Reply:
we are still doing it in a write transaction. The Write transaction gets set up in createProject(), and it passes the transaction context into this function